site/docs/encoding/condition.md

```diff
@@ -8,9 +8,11 @@ For [mark property channels](encoding.html#mark-prop) as well as [text and toolt
 
 There are two ways to use the `condition` property:
 
-1) Combining a [conditional field definition](#field) with a value definition
+1) Combining a [conditional field definition](#field) with a value definition.
+
+2) Combining a [conditional value definition](#value) or an array of conditional value definitions with a  field definition or a value definition.
```

Comment:
This wording is confusing. Here's what I'd suggest:

```markdown
1. Combining one conditional field definition with one value definition.
2. Combining one or more conditional value definitions with a single field or value definition. 
```

I'd favor some brevity here since "conditional [blah] definition" is already a mouthful (I tried to brainstorm alternatives here but couldn't come up with anything better... maybe we drop "definition"?). We link to the two cases anyways, so the fact that (2) should be expressed as an array will be made explicit later on.